tests/llmcompressor/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -301,12 +305,14 @@ def test_model_shared_tensors(
     offload, torch_dtype, tie_word_embeddings, device_map, tmp_path
 ):
     # load model
-    model = SparseAutoModelForCausalLM.from_pretrained(
+    model = AutoModelForCausalLM.from_pretrained(
         "Xenova/llama2.c-stories15M",
         torch_dtype=torch_dtype,
         tie_word_embeddings=tie_word_embeddings,
         device_map=device_map,
     )
+    patch_tied_tensors_bug(model)
```

Comment:
Why do we now need this patch?

Reply:
SparseAutoModelForCausalLM on load ran this code before